Travel Scheduler/View Controllers/FirstScreenViewController.m

```diff
@@ -87,8 +90,53 @@ - (void)viewDidLoad
     [self createAutocompleteTableView];
     self.autocompleteTableView.delegate = self;
     self.autocompleteTableView.dataSource = self;
+    [self makeAnimatedLine];
+    [self setUpImage];
+    
+    
+    for(NSArray *family in UIFont.familyNames) {
+        for(NSString *name in [UIFont fontNamesForFamilyName:family]) {
+            NSLog(@"%@", name);
+        }
```

Comment:
Is this testing stuff?

Reply:
lol yeah, i was trying to find the correct name for the font I wanted. Gonna erase that, thanks for pointing out!